test/java/src/com/ibm/streamsx/topology/test/api/LowLatencyTest.java

```diff
@@ -90,7 +90,7 @@ public void threadedPortTest() throws Exception{
                 lowLatencyTag = (String) placement.get(JOperator.PLACEMENT_LOW_LATENCY_REGION_ID);
             String kind = (String)op.get("kind");
             JSONObject queue = (JSONObject) op.get("queue");
-            if(queue != null && (lowLatencyTag!=null || lowLatencyTag.equals(""))){
+            if(queue != null && (lowLatencyTag == null || lowLatencyTag.equals(""))){
```

Comment:
I assume this was changed to a code not reached warning, but not sure if the replacement makes much sense either.